pages/dkp/kommander/1.4/release-notes/1.4.3/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.4.3
+title: Release Notes Kommander 1.4.3
+menuWeight: 40
+excerpt: View release-specific information for Kommander 1.4.3
+enterprise: false
+---
+
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Kommander&reg; version 1.4.3 was released on December 9, 2021.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/1.8/download/) and [install](/dkp/konvoy/1.8/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+## Features/Improvements
+
+- Updated Kubecost enterprise key
```

Comment:
Does this have any user visibility or impact?   If not, I'd argue this is not really a feature or improvement.  If it does, we should focus on what 'feature/improvement' this adds to the user experience. 

@gracedo do you know?

Reply:
agree there.

Reply:
@cneth @cmastr I found the related ticket: https://jira.d2iq.com/browse/D2IQ-81160?jql=text%20~%20%22Kubecost%20enterprise%20key%22

Is the legacy userbase wide enough to include as @gracedo mentioned: "For legacy users using enterprise features of kubecost, we need to come up with a plan. Perhaps we can first get the key to unblock the 2.1 release, then invalidate the old key once we have a plan. It likely involves asking customers to upload the new key - or upgrading to a patch release."

Reply:
No cops here. Old key is still valid, I don't think we need to include any messaging for this now.

Reply:
```suggestion
## Features/Improvements

- Updated Kubecost enterprise key.
```